tests/public_models/test_public_models_predicts.py

```diff
@@ -47,6 +49,32 @@ def test_audio_predict_on_public_models(channel):
         )
 
 
+@both_channels
+def test_text_gen_on_public_llms(channel):
+    """Test text generation for public large language models.
+    All these models can take the same test text prompt.
+    """
+    stub = service_pb2_grpc.V2Stub(channel)
+
+    for title, model_id, app_id, user_id in LLM_TITLE_ID_TUPLES:
+        request = service_pb2.PostModelOutputsRequest(
+            user_app_id=resources_pb2.UserAppIDSet(user_id=user_id, app_id=app_id),
+            model_id=model_id,
+            inputs=[
+                resources_pb2.Input(
+                    data=resources_pb2.Data(text=resources_pb2.Text(raw=TEXT_GEN_PROMPT))
+                )
+            ],
+        )
+        response = post_model_outputs_and_maybe_allow_retries(
```

Comment:
We're going to have to update post_model_outputs_and_maybe_allow_retries to retry even in prod, and number of retries will have to go up since these LLM's can take really long to load. Also we should probably only add one hosted LLM rather than three, otherwise this test will add way too much time to our pipelines and be flakier.